ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/af-commons/v1/clientlibs/editor/utils/utils.js

```diff
@@ -167,25 +167,26 @@
             _managePatternDynamicBehaviour();
             patternComponent.addEventListener("change", _managePatternDynamicBehaviour );
             function _managePatternDynamicBehaviour() {
-                var displayPatternSelectedValue = patternComponent.selectedItem.innerHTML;
+                // FORMS-12822, below the pattern was compared based on the name rather than the value ("No Pattern" instead of ####.####) which was creating issue in other languages therefore now it has changed to value.
```

Comment:
Please don't add product JIRA in open source code, can you remove it.

Reply:
removed, please review.